src/index.html

```diff
@@ -1,13 +1,80 @@
 <!doctype html>
+
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
+    >
   </head>
-  <body>
-    <h1>Product cards</h1>
+
+  <body class="page">
+    <div
+      class="page-cont"
+      data-qa="card"
+    >
+
+      <img
+        src="./images/imac.jpeg"
+        alt="iMac"
+        class="page-cont__img"
+      >
+
+      <h2 class="page-cont__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="page-cont__prod-code">
+        Product code: 195434
+      </p>
+
+      <div class="page-cont__review-cont">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="page-cont__review-count">
+          Reviews: 5
+        </div>
+      </div>
+
+      <div class="page-cont__prod-price">
+        <div class="page-cont__price-title">
+          Price:
+        </div>
+
+        <div class="page-cont__price-cur">
+          $2,199
+        </div>
+      </div>
+
+      <a
+        href="#"
+        class="page-cont__button-buy"
+        data-qa="hover"
+      >BUY</a>
```

Comment:
code styling is broken
```suggestion
      <a
        href="#"
        class="page-cont__button-buy"
        data-qa="hover"
      >
        BUY
      </a>
```